galsim/bandpass.py

```diff
@@ -458,18 +458,31 @@ def thin(self, rel_err=1.e-4, preserve_range=False):
 
         @param rel_err            The relative error allowed in the integral over the throughput
                                   function. [default: 1.e-4]
+        @param trim_zeros         Remove redundant leading and trailing points where f=0?  (The last
+                                  leading point with f=0 and the first trailing point with f=0 will
+                                  be retained).  Note that if both trim_leading_zeros and
+                                  preserve_range are True, then the only the range of `x` *after*
+                                  zero trimming is preserved.  [default: True]
         @param preserve_range     Should the original range (`blue_limit` and `red_limit`) of the
                                   Bandpass be preserved? (True) Or should the ends be trimmed to
                                   include only the region where the integral is significant? (False)
-                                  [default: False]
+                                  [default: True]
+        @param fast_search        If set to True, then the underlying algorithm will use a
+                                  relatively fast O(N) algorithm to select points to include in the
+                                  thinned approximation.  If set to False, then a slower O(N^2)
+                                  algorithm will be used.  The slower algorithm usually yields a
+                                  slightly more optimal thinned representation of the original
```

Comment:
>more optimal = one that gives more accurate estimates of fluxes in bandpasses, or ...?

How about: 

We have found that the slower algorithm tends to yield a thinned representation that retains fewer samples while still meeting the relative error requirement, and may also be somewhat more robust when computing SED fluxes through Bandpasses when a significant fraction of the integrated flux passes through low throughput bandpass light leaks.